### ISSUE TITLE
Custom values in windi.config.js do not show up as suggestions

### ISSUE THREAD

**Issue by vexkiddy** [windicss/windicss-intellisense#295]
Consider this windi.config.js file

```
import { defineConfig } from 'windicss/helpers';
import colors from 'windicss/colors';

export default defineConfig({
  darkMode: 'class', // or 'media'
  theme: {

    colors: {
        color1: "#33FF9D",
        color2: "#33FF9D",
        color3: "#33FF9D",
        color4: "#33FF9D",
      },
      fontFamily: {
        semibold: ['SofiaSemiBold', 'sans-serif'],
        light: ['SofiaLight', 'sans-serif'],
        bold: ['SofiaBold', 'sans-serif'],
        regular: ['SofiaRegular', 'sans-serif'],
        medium: ['SofiaMedium', 'sans-serif']

      },
  },
})
```

This should overwrite the default colours from windi, which it does and they do work in production. However the plugin still suggests the original windicss colours and doesn't suggest these new ones. Any ideas why that is ?

**Comment by devmor-j** [comment]
I have the same problem :|

**Comment by timmychatterbug** [comment]
yep.. its really annoying :D

**Comment by alexanderniebuhr** [comment]
@vexkiddy @devmor-j @timmychatterbug  I can not reproduce this. It works perfectly fine.
Can you test this with the latest pre-release version `v0.23.4`?

**Comment by devmor-j** [comment]
Yeah, for reproduction you can test this repo and I put the instructions, (it's a simple card component):
https://github.com/devmor-j/fm-order-summary-component

I disabled all other css snippets and tailwind intellisense as well. First time you open project and use custom colors like `clr-pale-blue` It works.

here is how to reproduce the issue (not showing new custom colors - maybe even all custom configs):

**Instructions to reproduce issue - custom color not showing up**
1. make sure the `WindiCSS IntelliSense` is enabled
2. go to file `windi.config.js` then find *colors* part
3. add a new color at the end of the list (bright purple), ex:
`'clr-green': 'hsl(145, 50%, 50%)', // Green Color`
4. save config and go to `./index.html`
5. use that color for a element like `h1` remove current color `clr-dark-blue` and add yours (`w:text="clr-green"` or `class="text-clr-green"`)
6. **no suggestion appears** (but dev server works fine and color is changed)
7. to run vite dev server use `npm run dev` or `pnpm dev`
8. the `h1` element should be green (or any color you set in config)

**What I thing might be the issue**

Seems like configs are only read when the windicss extension is loaded. After that any changes in config is not added to suggestion engine.

**Here is how to fix this**

1. Just **disable** windicss extension and **reload vscode**
2. Then **enable** extension again (without changing `windi.config.js`)
3. The new custom color will show up in suggestions even though the suggestion icon is missing (shows an *art board* object next to it while other native windicss colors have *cube colors*).
4. If you add a new color in `windi.config.js` the same problem repeats and you have to do instructions 1-3 to fix this.

Hope this helps and thanks for your response.

**Comment by mariohamann** [comment]
Hey!

Wanted to experiment with WindiCSS but experienced the same issues as mentioned above and therefore stopped working with it. 😢 (Mac, M1 Pro)

I experienced another strange problem, maybe that helps you with debugging:
I had two projects open at the same time in two different windows, both using WindiCSS. Most of the time added colors etc. didn't show up, even after reloading the window – strange enough, at one point I saw (old) suggestions from the project of the other window, but not the suggestions of the current project.

It feels like some strange caching behaviour or similar...?

**Comment by alexanderniebuhr** [comment]
Yeah. It's a caching issue. I've tried to implement a new loading logic in the newest pre-release version v0.23.5. Hope that helps. Will see.

**Comment by ckvv** [comment]
@alexanderniebuhr 
I still have a similar problem using `v0.23.5` in vs. After modifying the `alias config`, the style must be restarted to take effect

**Comment by devmor-j** [comment]
Seems like this issue can be closed as of v0.23.5 (pre-release version) works fine.
Tested with *Vite* not only reloads config file on change, but also suggestions appear as expected.
Thanks @alexanderniebuhr 😀

**Comment by SaulAbreu** [comment]
@devmor-j @alexanderniebuhr how did you solve this error? How can I implement a new loading logic in the newest pre-release version v0.23.5?
